src/main/java/Duke.java

```diff
@@ -1,10 +1,186 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String inputStatement;
+        int endFlag = 0;
```

Comment:
I think endFlag should be a Boolean and it should be named Boolean hasEnded = false instead, to indicate that the loop has exited.